libwacom/libwacom.h

```diff
@@ -636,6 +637,33 @@ uint32_t libwacom_match_get_product_id(const WacomMatch *match);
 uint32_t libwacom_match_get_vendor_id(const WacomMatch *match);
 const char* libwacom_match_get_match_string(const WacomMatch *match);
 
+/**
+ * Return true if the given manufacturer_id matches those defined for this
+ * device.
+ */
+bool libwacom_edid_match_manufacturer_id(WacomDevice *device,
+					 const char *manufacturer_id);
+
+/**
+ * Return true if the given product code matches those defined for this
+ * device.
+ */
+bool libwacom_edid_match_product_code(WacomDevice *device,
+				      uint16_t product_code);
+
+/**
+ * Return true if the given serial number matches those defined for this
+ * device.
+ */
+bool libwacom_edid_match_serial_number(WacomDevice *device,
+				       uint32_t serial_number);
+
+/**
+ * Return true if the given EDID name matches those defined for this device.
+ */
+bool libwacom_edid_match_product_name(WacomDevice *device,
+				      const char *product_name);
+
```

Comment:
I'm not sure I like the idea of matching each part of the EDID in isolation. It seems like you'd potentially wind up with false positives. E.g. a device with `EdidMatch=ABC:1234:*:*;XYZ:9876:*:*` could be matched by monitors with EDID `ABC:9876:*:*` or `XYZ:1234:*:*`.

Reply:
I've changed this to a custom `WacomEDID` struct that is set by the caller and then compared as a whole in the latest push